addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -333,6 +302,74 @@
                 </div>
             </div>
         </t>
+        <t t-elif="channel.enroll in ['public', 'invite']" name="o_wslides_prerequisite_test">
+            <div t-if="channel.prerequisite_channel_ids and not channel.has_completed_prerequisites"
+                 class="text-center">
+                <div class="alert my-0 bg-100">
+                    <h6>
+                        <i class="fa fa-lock"/>
+                        <span>Course Locked</span>
+                    </h6>
+                    <div>
+                        <small>
+                            Finish
+                            <a t-if="len(channel.prerequisite_channel_ids) == 1"
```

Comment:
Same question about opening in a tab for every links, maybe that is not what we want to do, to double-check with François.